schemas/blocks/summary.json

```diff
@@ -12,6 +12,10 @@
           "Summary",
           "SectionSummary"
         ]
+      },
+      "collapsible": {
+        "description": "Check whether the summary page contains a collapsible list of section summaries",
```

Comment:
A better description might be `"When True, the summary page will contain a list of collapsed section summaries"`